app/Console/Commands/ImportAshesCampaigns.php

```diff
@@ -0,0 +1,115 @@
+<?php
+
+namespace Rogue\Console\Commands;
+
+use Carbon\Carbon;
+use League\Csv\Reader;
+use Rogue\Models\Campaign;
+use Rogue\Services\Registrar;
+use Illuminate\Console\Command;
+
+class ImportAshesCampaigns extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'rogue:ashescampaignsimport {path}';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Imports legacy campaigns from Ashes based on the data in the given (link to a) CSV file.';
+
+    /**
+     * Create a new command instance.
+     *
+     * @return void
+     */
+    public function __construct(Registrar $registrar)
+    {
+        parent::__construct();
+    }
+
+    /**
+     * Execute the console command.
+     *
+     * @return mixed
+     */
+    public function handle()
+    {
+        // Make a local copy of the CSV
+        $path = $this->argument('path');
+        info('rogue:legacycampaignimport: Loading in csv from ' . $path);
+
+        $temp = tempnam(sys_get_temp_dir(), 'command_csv');
+        file_put_contents($temp, fopen($this->argument('path'), 'r'));
+
+        // Load the legacy campaigns from the CSV
+        $legacy_campaigns_csv = Reader::createFromPath($temp, 'r');
+        $legacy_campaigns_csv->setHeaderOffset(0);
+        $legacy_campaigns = iterator_to_array($legacy_campaigns_csv->getRecords());
+
+        // Import each legacy campaign
+        info('rogue:legacycampaignimport: Importing legacy campaigns...');
+
+        foreach ($legacy_campaigns as $iterator => $legacy_campaign) {
+            // Normalize all "NULL" values to null
+            foreach ($legacy_campaign as $key => $value) {
+                if ($value === "NULL") {
+                    $legacy_campaign[$key] = null;
+                }
+            }
+
+            // See if the campaign exists
+            $existing_campaign = Campaign::where('id', $legacy_campaign['campaign_id'])->orWhere('id', $legacy_campaign['run_id'])->first();
```

Comment:
@DFurnes great q! Yes, this should work because the way we write the query and export the results to a CSV from Ashes, it'll always be reverse order of old runs -> current like you said. 

Here's an example!

![screen shot 2018-11-28 at 1 22 05 pm](https://user-images.githubusercontent.com/9019452/49173082-b1094080-f310-11e8-81cf-21c546630512.png)